trace/trace_test.go

```diff
@@ -65,24 +65,25 @@ func checkChild(p SpanContext, c *Span) error {
 	if c == nil {
 		return fmt.Errorf("got nil child span, want non-nil")
 	}
-	if got, want := c.spanContext.TraceID, p.TraceID; got != want {
+	if got, want := c.SpanContext().TraceID, p.TraceID; got != want {
```

Comment:
Isn't this a breaking change still?

Reply:
spanContext is a non-exported field, so the removal of it from the `Span` struct isn't breaking for anyone using it as a library.